Source/Core/Core/CoreTiming.h

```diff
@@ -68,6 +69,8 @@ enum class FromThread
 };
 
 // userdata MAY NOT CONTAIN POINTERS. userdata might get written and reloaded from savestates.
+// After the first Advance, the slice lengths and the downcount will be reduced whenever an event
+// is scheduled earlier than the current values (when scheduled from the CPU Thread only).
```

Comment:
But only when `ScheduleEvent` hasn't been called from inside the `Advance` call. 

The reasoning behind `Init`'s behavior, was: Any events that get scheduled between `Init` and the zeroth cycle should be treated the same as if they were scheduled inside an `Advance` (aka, inside the slice boundary, considering the first slice hasn't started yet).